docs/Basics-Layout.md

```diff
@@ -1,17 +1,17 @@
 ---
 id: basics-layout
-title: Layout
+title: Layout with Flexbox
 layout: docs
 category: The Basics
-permalink: docs/basics-layout.html
+permalink: docs/flexbox.html
```

Comment:
I mean that docs/flexbox.html is popular. docs/basics-layout.html doesnt exist yet. Previously docs/flexbox.html was the autogen flexbox doc, now it's this part of "the basics", so the links should just send people here now.